packages/circuits/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@unirep/circuits",
+  "version": "1.0.4",
+  "main": "circuits/utils.js",
+  "scripts": {
+    "build": "yarn downloadPtau && yarn buildCircuits && yarn buildVerifyEpochKeySnark && yarn buildUserStateTransitionSnark && yarn buildProveReputationSnark && yarn buildProveUserSignUpSnark && tsc",
+    "downloadPtau": "./scripts/downloadPtau.sh",
+    "buildCircuits": "npx ts-node scripts/buildCircuits.ts",
+    "buildVerifyEpochKeySnark": "./scripts/buildVerifyEpochKeySnark.sh",
+    "buildUserStateTransitionSnark": "./scripts/buildUserStateTransitionSnark.sh",
+    "buildProveReputationSnark": "./scripts/buildProveReputationSnark.sh",
+    "buildProveUserSignUpSnark": "./scripts/buildProveUserSignUpSnark.sh",
+    "test": "mocha -r ts-node/register test/*.ts --exit"
+  },
+  "repository": "https://github.com/Unirep/circuits.git",
+  "author": "vivianjeng <vivi432@yahoo.com.tw>",
+  "license": "MIT",
+  "dependencies": {
+    "@unirep/crypto": "git+https://github.com/Unirep/crypto.git",
```

Comment:
Make sure to put a version number here so Lerna will link the packages locally on `bootstrap`

Reply:
Thanks @vimwitch! I update the PR.